src/OmniSharp.Host/Startup.cs

```diff
@@ -20,6 +20,7 @@
 using OmniSharp.Services.FileWatching;
 using OmniSharp.Stdio.Logging;
 using OmniSharp.Stdio.Services;
+using System.IO;
```

Comment:
yes, I deleted it now as it wasn't even used; it crept in here because of some code that I ended up ultimately removing